.zuul.yaml

```diff
@@ -1,6 +1,60 @@
 ---
+- job:
+    name: stf-base
+    # defined in: https://review.rdoproject.org/cgit/config/tree/zuul.d/_jobs-crc.yaml
+    parent: base-simple-crc
+    # need to set cluster-monitoring
+    abstract: true
+    description: |
+      Create a CRC instance that's configured for STF
+    # irrelevant-files
+    roles:  # adds in dependent roles i.e. put it in the role path
+      - zuul: github.com/openstack-k8s-operators/ci-framework
+    # These are the additional repos that zuul will clone
+    required-projects:
+      - openstack-k8s-operators/ci-framework
```

Comment:
so you need to do a branch overide here since this uses main

https://github.com/rdo-infra/review.rdoproject.org-config/blob/7a8d010cf5fdedb1c74572722cd6d4552de4e44f/zuul.d/openstack-ci-base.yaml#L52-L54

    required-projects:
      - name: github.com/openstack-k8s-operators/ci-framework
        override-checkout: main